### PR TITLE
feat(README): add more badges in the README

### PR DESCRIPTION
**Pull request by adelkahomolova**
Add badges in the README

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,11 +26,13 @@
     "@oclif/config": "^1",
     "@oclif/plugin-help": "^2",
     "axios": "^0.19.0",
+    "codecov": "^3.5.0",
```
Comment (prokopsimek) [inline package.json]: This lib should be DEV dependency.

File: package.json [hunk 2]
```diff
@@ -26,11 +26,13 @@
...
     "colors": "^1.3.3",
     "debug": "^4.1.1",
     "git-url-parse": "^11.1.2",
     "glob": "^7.1.4",
     "inversify": "^5.0.1",
+    "istanbul": "^0.4.5",
```
Comment (prokopsimek) [inline package.json]: This lib should be DEV dependency.

File: package.json [hunk 3]
```diff
@@ -89,5 +92,9 @@
     "developer experience",
     "dxheroes",
     "code analysis"
-  ]
+  ],
+  "jest": {
```
Comment (prokopsimek) [inline package.json]: Is it necessary to have this config here if we have `jest.config.js` already?

File: .travis.yml [hunk 4]
```diff
@@ -9,11 +9,19 @@ cache:
...
 
+install:
+  - yarn add codecov
+
 script:
   - yarn build
   - yarn test --coverage
   - yarn lint
   - bin/run -v
+  - yarn add codecov
```
Comment (prokopsimek) [inline .travis.yml]: It's not necessary since we install the library with other dependencies.

File: .travis.yml [hunk 5]
```diff
@@ -9,11 +9,19 @@ cache:
     - lib
     - node_modules
 
+install:
+  - yarn add codecov
```
Comment (prokopsimek) [inline .travis.yml]: Duplicity with `:20`

File: .travis.yml [hunk 6]
```diff
@@ -9,11 +9,19 @@ cache:
...
+
 script:
   - yarn build
   - yarn test --coverage
   - yarn lint
   - bin/run -v
+  - yarn add codecov
+  - yarn codecov -t 880f291b-c196-4388-bffa-a4b92c2eac32
```
Comment (prokopsimek) [inline .travis.yml]: Token is not required for public repos.

File: package.json [hunk 7]
```diff
@@ -62,6 +64,7 @@
     "eslint-plugin-prettier": "^3.1.0",
     "jest": "^24.8.0",
     "nock": "^10.0.6",
+    "nyc": "^14.1.1",
```
Comment (prokopsimek) [inline package.json]: Remove

File: package.json (hunk 2 above)
Comment (adelkahomolova) [reply package.json]: It's no longer used, I'll remove it.

File: package.json (hunk 3 above)
Comment (adelkahomolova) [reply package.json]: It shouldn't, I'll remove it.

File: .travis.yml (hunk 4 above)
Comment (adelkahomolova) [reply .travis.yml]: Ok, I'll remove it.

File: .travis.yml (hunk 5 above)
Comment (adelkahomolova) [reply .travis.yml]: Ok, I'll remove it.

File: .travis.yml (hunk 6 above)
Comment (adelkahomolova) [reply .travis.yml]: Ok, I'll remove it.

File: package.json (hunk 1 above)
Comment (adelkahomolova) [reply package.json]: So should I move it the devDependencies?

**Comment by adelkahomolova** [comment]
@all-contributors bot please add @adelkahomolova for code

File: package.json (hunk 1 above)
Comment (prokopsimek) [reply package.json]: yep

**Pull request merged**